src/Symfony/Component/Messenger/HandleTrait.php

```diff
@@ -46,9 +48,9 @@ private function handle(object $message): mixed
         }
 
         if (\count($handledStamps) > 1) {
-            $handlers = implode(', ', array_map(fn (HandledStamp $stamp): string => \sprintf('"%s"', $stamp->getHandlerName()), $handledStamps));
+            $handlers = implode('", "', array_map(fn (HandledStamp $stamp): string => $stamp->getHandlerName(), $handledStamps));
 
-            throw new LogicException(\sprintf('Message of type "%s" was handled multiple times. Only one handler is expected when using "%s::%s()", got %d: %s.', get_debug_type($envelope->getMessage()), static::class, __FUNCTION__, \count($handledStamps), $handlers));
+            throw new LogicException(\sprintf('Message of type "%s" was handled multiple times. Only one handler is expected when using "%s::%s()", got %d: "%s".', get_debug_type($envelope->getMessage()), static::class, __FUNCTION__, \count($handledStamps), $handlers));
```

Comment:
Fix for fabbot error. Output is kept the same but this way fabbot does not yell.

Reply:
I think we should revert and ignorer fabbot here

Reply:
Removed that commit.